braze/src/main/java/com/rudderstack/android/integration/braze/BrazeIntegrationFactory.java

```diff
@@ -639,4 +654,24 @@ private static boolean getBoolean(Object value) {
         }
         return false;
     }
+
+    private ConnectionMode getConnectionMode(Map<String, Object> config) {
+        String connectionMode = (config.containsKey(CONNECTION_MODE)) ? (String)config.get(CONNECTION_MODE) : null;
+        if (connectionMode == null) {
+            if (config.containsKey(USE_NATIVE_SDK_TO_SEND) && getBoolean(config.get(USE_NATIVE_SDK_TO_SEND))) {
```

Comment:
Remove this logic on USE_NATIVE_SDK_TO_SEND & in case of connectionMode being null, default will be cloud